strawberry/arguments.py

```diff
@@ -26,10 +39,28 @@ def get_arguments_from_annotations(
             origin_name=name,
             name=to_camel_case(name),
             origin=origin,
-            type=annotation,
             default_value=default_value,
         )
 
+        if get_origin(annotation) is Annotated:
+            argument_definition.type = get_args(annotation)[0]
+
+            argument_metadata = None
+            # Find any instances of StrawberryArgument in the Annotated metadata
+            # raising an exception if there are multiple StrawberryArguments
+            for arg in get_args(annotation)[1:]:
```

Comment:
can we store `get_args(annotation)` in a variable?

Reply:
also I'm not sure I understand this whole check, sorry, I'm having a slow day :D

Reply:
> can we store get_args(annotation) in a variable?

Yep can do (I was just being lazy because I couldn't think of a good name for it!)

> also I'm not sure I understand this whole check, sorry, I'm having a slow day :D

What part are you having problems understanding?